cfgov/v1/management/commands/update_data_snapshot_values.py

```diff
@@ -0,0 +1,67 @@
+import ast
+
+from django.core.management.base import BaseCommand
+
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_changes
+
+
+class Command(BaseCommand):
+    help = 'Monthly updates to data snapshot values'
+
+    def add_arguments(self, parser):
+        """Adds all arguments to be processed."""
+        parser.add_argument(
+            '--markets',
+            nargs='?',
+            help='String representation of a list of markets'
+        )
+
+    def get_data_snapshots(self):
+        """ Gets all data snapshots from browse pages
+        Assumes there is one data snapshot per page
+        """
+        snapshots = []
+        for page in BrowsePage.objects.all():
```

Comment:
Somewhat offtopic: I'm a bit wary of this design because accessing `page.specific.content.stream_data['type']` is a bit of a hidden/undocumented Wagtail thing and therefore potentially brittle.

Did you consider using a snippet type for this (like [`Contact`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/snippets.py#L47)), or maybe a Django model that can be embedded on a page (like [`GradePanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jobmanager/models/panels.py#L57))? That would provide a centralized place for editors to modify the data being embedded on pages, and still let one or more pages easily embed the data in the form of a block.

You also wouldn't need to republish the page once the data model gets updated, as it should automatically update when the underlying data does (caching aside).